trie/verkle.go

```diff
@@ -231,11 +231,6 @@ func (trie *VerkleTrie) Hash() common.Hash {
 	return trie.root.Commit().Bytes()
 }
 
-func nodeToDBKey(n verkle.VerkleNode) []byte {
-	ret := n.Commitment().Bytes()
-	return ret[:]
-}
-
```

Comment:
this function is no longer in use, as it was still using the "hash-based" storage model